src/services/housing.js

```diff
@@ -5,7 +5,7 @@
  */
 
 import http from './http';
-import './user'; // Needed for typedef of StudentProfileInfo
+import './user';
```

Comment:
```suggestion
import './user'; // Needed for typedef of StudentProfileInfo
```
Add this comment back in, it was removed by mistake